src/lib/connectors/repos/billing-batches.js

```diff
@@ -113,40 +113,38 @@ const create = async data => {
   return model.toJSON();
 };
 
-const findSentTPTBatchesForFinancialYearAndRegion = async (financialYear, regionId) => {
+const findSentBatchesForRegion = async (regionId, optionalFilters = {}, withRelated = ['region']) => {
   const filters = {
     status: BATCH_STATUS.sent,
-    batch_type: BATCH_TYPE.twoPartTariff,
-    to_financial_year_ending: financialYear,
-    region_id: regionId
+    region_id: regionId,
+    ...optionalFilters
   };
+
   const batches = await BillingBatch
     .forge()
     .where(filters)
-    .fetchAll({
-      withRelated: [
-        'billingInvoices',
-        'billingInvoices.billingInvoiceLicences',
-        'billingInvoices.billingInvoiceLicences.licence'
-      ]
-    });
+    .fetchAll({ withRelated });
 
   return batches.toJSON();
 };
 
-const findSentTPTBatchesForRegion = async regionId => {
+const findSentTPTBatchesForFinancialYearAndRegion = async (financialYear, regionId) => {
   const filters = {
-    status: BATCH_STATUS.sent,
     batch_type: BATCH_TYPE.twoPartTariff,
-    region_id: regionId
+    to_financial_year_ending: financialYear
   };
-  const batches = await BillingBatch
-    .forge()
-    .where(filters)
-    .fetchAll();
-  return batches.toJSON();
+  const withRelated = [
+    'billingInvoices',
+    'billingInvoices.billingInvoiceLicences',
+    'billingInvoices.billingInvoiceLicences.licence'
+  ];
+
+  return findSentBatchesForRegion(regionId, filters, withRelated);
 };
 
+const findSentTPTBatchesForRegion = async regionId =>
+  findSentBatchesForRegion(regionId, { batch_type: BATCH_TYPE.twoPartTariff });
```

Comment:
I wonder if the methods in this file could be simpler by making use of the `findMany` repo helper